core/build.gradle

```diff
@@ -70,8 +70,8 @@ project.tasks.check.dependsOn(jarFileTest)
 
 dependencies {
     compile 'junit:junit:4.12'
-    compile 'org.slf4j:slf4j-api:1.7.25'
-    compile 'org.slf4j:slf4j-ext:1.7.25'
+    compile 'org.slf4j:slf4j-api:1.8.0-beta2'
+    compile 'org.slf4j:slf4j-ext:1.8.0-beta2'
```

Comment:
I kinda feel bad about depending on beta2 version of it. Do you know if they will release a hotfix?